__tests__/integration/mirador/window_actions.test.js

```diff
@@ -19,5 +19,5 @@ describe('Window actions', () => {
     )); // only default configed windows found
     await page.waitFor(1000);
     await expect(numWindows).toBe(2);
-  });
+  }, 10000);
```

Comment:
Was this needed for the replacement of the iframe component? Were the tests failing?

Reply:
no, nothing to do with the component.  these puppeteer tests consistently fail for me locally and also sometimes on CI.  increasing the timeout solves it.